support-frontend/app/services/PaymentAPIService.scala

```diff
@@ -100,12 +97,11 @@ class PaymentAPIService(wsClient: WSClient, paymentAPIUrl: String)(implicit ec:
   def executePaypalPayment(
     paymentJSON: JsObject,
     acquisitionData: JsValue,
-    queryStrings: Map[String, Seq[String]],
-    email: Option[String],
+    email: String,
     isTestUser: Boolean,
     userAgent: Option[String]
   )(implicit ec: ExecutionContext): EitherT[Future, PaymentAPIResponseError[PayPalError], PayPalSuccess] = {
-    val data = ExecutePaymentBody(email, acquisitionData, paymentJSON)
-    postPaypalData(data, queryStrings, isTestUser, userAgent).subflatMap(decodePaymentAPIResponse[PayPalError, PayPalSuccess])
+    val data = ExecutePaymentBody(Some(email), email, acquisitionData, paymentJSON)
```

Comment:
Do we want this to accept `Some(email)` and `email` as args? My knowledge of Scala isn't extensive but I thought you could only call `Some` on an option, and email is now a string

Reply:
Ah I think I understand - Some(email) _returns_ an option here and this will also get removed after the Payment API switches over

Reply:
yep, a type of `Option[String]` has one of two forms, either `Some("some string")`, or `None`. To construct an `Option[T]` from a `T` you do `Some(thingOfTypeT)`